### ISSUE TITLE
Error: No executable gpg binary found.

### ISSUE THREAD

**Issue by mikebronner** [phar-io/phive#307]
I'm not sure if this error is caused by something I misconfigured, or by phive. When running the command

```sh
Bin/phive install phpcs --target Bin/ --copy
```

I get the following error:
```sh
Phive 0.14.5 - Copyright (C) 2015-2021 by Arne Blankerts, Sebastian Heuer and Contributors
[ERROR]    No executable gpg binary found.  
[ERROR]     Either install gpg or enable the gnupg extension in PHP. 
```

Running `which gig` produces the following, perhaps expected, output:
```
gpg not found
```

I am running on macOS 11.3 beta. I'll be happy to provide additional information, whatever helps. :) Thanks!

**Comment by theseer** [comment]
Well, `phive` requires the gnupg binary to be present to operate as it checks keys and signatures. I'm not sure how you installed `php` and `phive`. If you have homebrew installed, `brew install gnupg` should be of help.

**Comment by mikebronner** [comment]
Thanks @theseer. macOS comes with PHP 7.3 already installed, and I downloaded PHIVE from the GitHub repo. Unfortunately it seems that macOS does not come with any PGP executable installed.

I did get things working by installing "GnuPG for osx", but in the end I'm looking for a way to bundle PHIVE in an IDE extension that doesn't rely on external dependencies. I'll play around with providing the binary in the extension and referencing it when running PHIVE.

Thanks for responding, I'll close this issue for now. I was surprised by the requirement for PGP, since I didn't recall requiring it just.a few months back, so I opened this issue.
